Cargo.lock

```diff
@@ -198,62 +256,59 @@ checksum = "baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd"
 
 [[package]]
 name = "clap"
-version = "4.1.14"
+version = "4.5.4"
```

Comment:
https://github.com/clap-rs/clap/blob/5e4facf76fb38e04edba6cb1ab67d4f2b1ec8550/Cargo.toml#L17

I don't think this really compiles on v1.70.0.

```console
console on  fork/dev:dev [$] via 🦀 v1.70.0 
❯ cargo r                           
error: package `clap v4.5.4` cannot be built because it requires rustc 1.74 or newer, while the currently active rustc version is 1.70.0
Either upgrade to rustc 1.74 or newer, or use
cargo update -p clap@4.5.4 --precise ver
where `ver` is the latest version of `clap` supporting rustc 1.70.0

```

Reply:
Yeah, you're right. Unless there's some great reason to avoid bumping the MSRV for this, it's at 1.74 now.

Reply:
Bumped in https://github.com/tokio-rs/console/pull/515/commits/25e1efcb6445828eaa950e5362ae0f83a4836c13